ister.py

```diff
@@ -412,7 +412,7 @@ def get_current_format():
 def copy_os(args, template, target_dir):
     """Wrapper for running install command
     """
-    LOG.info("Starting swupd. May take several minutes")
```

Comment:
Reason for this? Regardless it should not be in the same commit as a functional change.